tests/test_vxlan_vtep_vni.rb

```diff
@@ -49,6 +49,8 @@ def test_create_with_existing
   end
 
   def test_vnis
+    skip('Platform does not support vnis') if VxlanVtepVni.vnis['nve1'].nil?
```

Comment:
@robert-w-gries @glennmatthews.  Spoke with Rob this morning.  The amount of work needed is beyond the scope of what we need for release 1.2 which goes live on 1/29.  I am fine with doing this in develop but we need this code to collapse today so we can pull the release branch.  Any objection to this?